.eslintrc.yml

```diff
@@ -0,0 +1,7 @@
+root: true
+extends:
+  - "@socialgouv/eslint-config-recommended"
+
+rules:
+  prefer-const: error
+  no-console: off
```

Comment:
we don't want console (we only tolerate it on frontend but we should move them to sentry log)

Reply:
Hum the issue is that most of our files in the dataset folder use `console.log` to pipe the output into files with `>>`

Reply:
what if we add "root" eslint config in data where we override the use of console ?

Reply:
Make total sense !

Reply:
done !